src/shared/components/case-editor/case-edit-form/case-edit-form.html

```diff
@@ -1,15 +1,17 @@
 <ng-container *ngFor="let field of fields">
     <div ccdConditionalShow ccdLabelSubstitutor [caseField]="field" [formGroup]="formGroup" [eventFields]="eventFields">
       <ng-container [ngSwitch]="field | ccdIsReadOnlyAndNotCollection ">
-        <ccd-field-read *ngSwitchCase="true" [caseField]="field" [withLabel]="true"></ccd-field-read>
+        <ccd-field-read *ngSwitchCase="true" [caseField]="field" [eventFields]="eventFields" [withLabel]="true"></ccd-field-read>
```

Comment:
I think the initial name for the `eventFields` @Input was chosen because the ConditionalShowDirective was only to be used on case progression. I think it could be renamed now to `caseFields` as it creates confusion if the directive is used in the case view context. Let me know what you think?

Reply:
I already wanted to name it caseFields, but was worried it will not be backwards compatible if someone else is using the component already and we change the Input field name.

Reply:
Confirmed with Andrew that this change will not break the contract. Changed eventFields to caseFields.